studio/base_artifact_store.py

```diff
@@ -0,0 +1,11 @@
+class BaseArtifactStore(object):
+
+    def __init__(self):
+        self.storage_client = None
+
+    def set_storage_client(self, sclient):
+        self.storage_client = sclient
+
+    def get_storage_client(self):
+        return self.storage_client
```

Comment:
The main purpose of this base class in ArtifactStore hierarchy
is to keep service-specific connection client.
This way we can access it without mentioning actual class,
which helps to untangle import cycles mess.